streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStreamThread.java

```diff
@@ -247,8 +247,8 @@ void pollAndUpdate() {
 
         public void close() throws IOException {
             try {
-                consumer.close();
-            } catch (Exception e) {
+                globalConsumer.close();
+            } catch (final RuntimeException e) {
```

Comment:
Could you elaborate a bit on this change?

Reply:
As `close()` does not throw any checked exception, it think it's cleaner code to catch `RuntimeException`